tox.ini

```diff
@@ -33,3 +36,30 @@ deps =
     pre-commit >= 2.9.3
 commands =
     pre-commit run -a
+
+[testenv:packaging]
+description =
+    Do packaging/distribution. If tag is not present or PEP440 compliant upload to
+    PYPI could fail
+# `usedevelop = true` overrides `skip_install` instruction, it's unwanted
+usedevelop = false
+# don't install molecule itself in this env
+skip_install = true
+deps =
+    check-manifest
+    collective.checkdocs >= 0.2
+    pep517 >= 0.8.2
```

Comment:
You should use pypa/build instead because the CLI is getting removed from `pep517` (it was experimental and never recommended).